### ISSUE TITLE
[bug] Typo in help command for conan remote auth (Conan 2.0)

### ISSUE THREAD

**Issue by gpxricky** [conan-io/conan#13221]
Hi,

I've observed the following typo when running **conan remote auth --help**:
```
...
positional arguments:
  remote       Pattern of the remote/s to disable. The pattern uses 'fnmatch' style wildcards.
...
```

**disable** doesn't make sense here ;).

Best regards,
gpxricky

**Comment by czoido** [comment]
Hi @gpxricky,
Thanks a lot for reporting, we'll fix that in 2.0.1 release

**Comment by memsharded** [comment]
Closed by https://github.com/conan-io/conan/pull/13285 for 2.0.1
